airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/connector_metrics.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import json
+from collections import defaultdict
+from typing import Optional
+
+import sentry_sdk
+from dagster import OpExecutionContext, asset
+from google.cloud import storage
+from orchestrator.logging import sentry
+
+GROUP_NAME = "connector_metrics"
+
+
+@sentry_sdk.trace
+def _safe_read_gcs_file(gcs_blob: storage.Blob) -> Optional[str]:
+    """Read the connector metrics jsonl blob.
+
+    Args:
+        gcs_blob (storage.Blob): The blob.
+
+    Returns:
+        dict: The metrics.
+    """
+    if not gcs_blob.exists():
+        return None
+
+    return gcs_blob.download_as_string().decode("utf-8")
+
+
+def _convert_json_to_metrics_dict(jsonl_string: str) -> dict:
+    """Convert the jsonl string to a metrics dict."""
+    metrics_dict = defaultdict(dict)
+    jsonl_lines = jsonl_string.splitlines()
+    for line in jsonl_lines:
+        data = json.loads(line)
+        connector_data = data["_airbyte_data"]
+        connector_definition_id = connector_data["connector_definition_id"]
+        airbyte_platform = connector_data["airbyte_platform"]
```

Comment:
💅 `airbyte_platform` value is cloud or oss right? 
I know it's out of the scope of this PR but `deployment_mode` sounds more aligned with or usual way of differentiating oss from cloud.

Reply:
Fair! But in this case Im going to keep away from asking Alex to do more work. Data was nice enough to prioritize this work even though they are already so busy.